Python: suspicious use of os.kill on Windows

The "The Parameter is incorrect" suggests that the API is used incorrectly:
```
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "C:\Users\runneradmin\AppData\Roaming\Python\Python38\site-packages\_pytest\main.py", line 267, in wrap_session
INTERNALERROR>     config._do_configure()
Trying to launch D:\a\ResInsight/ResInsight/cmakebuild/ApplicationExeCode/ResInsight
INTERNALERROR>   File "C:\Users\runneradmin\AppData\Roaming\Python\Python38\site-packages\_pytest\config\__init__.py", line 1053, in _do_configure
INTERNALERROR>     self.hook.pytest_configure.call_historic(kwargs=dict(config=self))
Launching as console app
INTERNALERROR>   File "C:\Users\runneradmin\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 514, in call_historic
Portnumber file retry count :  60
INTERNALERROR>     res = self._hookexec(self.name, self._hookimpls, kwargs, False)
Unable to read port number. Launch failed.
INTERNALERROR>   File "C:\Users\runneradmin\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "C:\Users\runneradmin\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 113, in _multicall
INTERNALERROR>     raise exception.with_traceback(exception.__traceback__)
INTERNALERROR>   File "C:\Users\runneradmin\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "D:\a\ResInsight\ResInsight\GrpcInterface\Python\rips\tests\conftest.py", line 48, in pytest_configure
INTERNALERROR>     _rips_instance = rips.Instance.launch(console=console)
INTERNALERROR>   File "D:\a\ResInsight\ResInsight\GrpcInterface\Python\rips\tests\../..\rips\instance.py", line 159, in launch
INTERNALERROR>     Instance.__kill_process(pid)
INTERNALERROR>   File "D:\a\ResInsight\ResInsight\GrpcInterface\Python\rips\tests\../..\rips\instance.py", line 88, in __kill_process
INTERNALERROR>     os.kill(pid, signal.CTRL_C_EVENT)
INTERNALERROR> OSError: [WinError 87] The parameter is incorrect
Error: Process completed with exit code 1.
```

--- a/GrpcInterface/Python/rips/instance.py
+++ b/GrpcInterface/Python/rips/instance.py
@@ -91,8 +91,8 @@ class Instance:
         Kill the process with a given pid.
         """
         if hasattr(signal, "CTRL_C_EVENT"):
-            # windows does not have kill
-            os.kill(pid, signal.CTRL_C_EVENT)
+            # windows - use SIGTERM for process termination
+            os.kill(pid, signal.SIGTERM)
         else:
             # linux/unix
             os.kill(pid, signal.SIGKILL)

